src/vm/exceptionhandling.cpp

```diff
@@ -4424,7 +4428,11 @@ VOID UnwindManagedExceptionPass2(EXCEPTION_RECORD* exceptionRecord, CONTEXT* unw
                 dispatcherContext.FunctionEntry,
                 callerFrameContext,
                 &handlerData,
+#ifdef _ARM_
+                (PULONG)&establisherFrame,
```

Comment:
Downcasting, it's actually a 32/64 bit thing so I guess that's a more suitable ifdef.